Update main Readme (#30)
DevExpress/testcafe-examples: README.md

> > We are in the process of filling this repository with more examples.

Reviewer: ```suggestion
> We are in the process of adding more examples to this repository.
```

The repository includes the following examples:

* [Find Element By Trimmed Text](examples/find-element-by-trimmed-text)
* [Use XPath Selectors](examples/use-xpath-selectors)
* [Access Element Properties](examples/element-properties)
* [Iterate Over Table Rows](examples/iterate-over-table-rows)
* [Test \<select\> Elements](examples/test-select-elements)
* [Page Reload](examples/page-manipulation)
* [Extract Code to Helpers](examples/extract-code-to-helpers)
* [Create Data-Driven Tests](examples/create-data-driven-tests)
* [Use Page Model](examples/use-page-model)
* [Inject Custom Client Scripts](examples/client-scripts)
* [Import Third-Party Modules](examples/import-third-party-modules)
* [Mock Date](examples/mock-date)
* [Mock Geolocation API](examples/mock-geolocation-api)
* [Wait Until an Element Property Has a Specific Value](examples/wait-for-element-property-value)
* [Wait For File Download](examples/wait-for-file-download)
* [Access Element Properties](examples/element-properties)

> Dedicated examples either run in one of the browsers (Chrome or Firefox) or require additional launch arguments. Launch those examples separately.

Reviewer 2: I think we can come up with a better word than 'dedicated'. I don't think we can come up with a *good* word (because the idea to separate examples with arguments has nothing to do with semantics) but it should be better than 'dedicated'.

`examples-with-args`, `arg-dependent`, `config-sensitive`? A combination of those?

Author: > detached-examples

> Below are the examples that either run in one of the browsers (Chrome or Firefox) or require additional launch arguments. Launch those examples separately.

Reviewer: ```suggestion
You can run the examples below in Chrome or Firefox or use additional launch arguments for other browsers. You should launch examples separately.
```
